cmd/manager/main.go

```diff
@@ -311,3 +315,15 @@ type finalizerFunc func(ctx context.Context, obj client.Object) (crfinalizer.Res
 func (f finalizerFunc) Finalize(ctx context.Context, obj client.Object) (crfinalizer.Result, error) {
 	return f(ctx, obj)
 }
+
+func authFilePathIfPresent(logger logr.Logger) string {
```

Comment:
Nit: Could we name this something like `mustAuthFilePathIfPresent` or `authFilePathIfPresentOrDie`? I think that it would help someone reading through the code where this function is used to understand that if there is an error (unhandled error type in this case) that this function will exit the program.

Reply:
I see where you're coming from. 
....

not sure if I would understand what `mustAuthFilePathIfPresent` would mean if I didn't know the context....

`authFilePathIfPresentOrDie` sounds like it must find an auth file or panic if it's not present, which is not the case. 

So I think this one's not that easy 😅

Reply:
Actually, I could just leave comment on top that says unhandled error will cause is to exit with error code 1. Gets the job done.

Reply:
While writing the comment I realized I have to basically summarize in English words every single thing that the function is doing, just to get to the part about the error code 1, which seems superfluous 😅 

```
// authFilePathIfPresent returns the value of the constant
// authFilePath ("/etc/catalogd/auth.json") if the file is
// present, emptry string ("") otherwise, using os.Stat(filePath).
// If os.Stat returns an error that's not IsNotExist, the function
// causes the program to exit with error code 1
```

Reply:
If there is no reasonable way we can come up with to convey this as part of the function name that is fine. I won't hold the PR on this nit, just thought it worthwhile to call out